commands/audit/audit_test.go

```diff
@@ -0,0 +1,159 @@
+package audit
+
+import (
+	biutils "github.com/jfrog/build-info-go/utils"
+	coreTests "github.com/jfrog/jfrog-cli-core/v2/utils/tests"
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-cli-security/utils/xray/scangraph"
+	clientTests "github.com/jfrog/jfrog-client-go/utils/tests"
+	"github.com/jfrog/jfrog-client-go/xsc/services"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const minXrayVersionForJas = utils.EntitlementsMinVersion
```

Comment:
I don't think a const is needed here, you can just use `utils.EntitlementsMinVersion`